std/math/index.zig

```diff
@@ -246,30 +246,10 @@ test "math.max" {
 /// The result is a compile time constant.
 pub fn minInt(comptime T: type) @typeOf(42) {
```

Comment:
`@typeOf(42)` can just be `comptime_int` now.

Reply:
Thanks!